internal/pkg/template/templates/environment/cf.yml

```diff
@@ -642,6 +642,12 @@ Outputs:
     Description: The ID of the Copilot-managed EFS filesystem.
     Export:
       Name: !Sub ${AWS::StackName}-FilesystemID
+{{- if .CDNConfig}}
+  CloudFrontDomainName:
+    Condition: CreateALB
```

Comment:
aren't we only using cf if a customer enables it for their environment? i feel like the condition `CreateALB` doesn't quite fit for that, but maybe i'm missing something 🤔

Reply:
CloudFront is an environment resource, but there's no reason to create it until there's an existing ALB to put it in front of, which is dictated by a deploy service. That's we we have the check `{{- if .CDNConfig}}` to check if CF is enabled, and then the condition `CreateALB` to make sure the resource is then only created once it's needed. Only when both of those are true will the resource actually be created.

Reply:
oh 🤦 i saw that 🙄